src/main/java/christmas/domain/consumer/VisitDay.java

```diff
@@ -0,0 +1,63 @@
+package christmas.domain.consumer;
+
+import christmas.domain.promotion.constants.SpecialPromotionPeriod;
+import christmas.exception.ExceptionHandler;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+
+import static christmas.domain.consumer.constants.PlannerConstraint.PROMOTION_MONTH;
+import static christmas.domain.consumer.constants.PlannerConstraint.PROMOTION_YEAR;
+
+public class VisitDay {
+    private static final int THURSDAY_VALUE = 4;
+    private static final int SUNDAY_VALUE = 7;
+
+    private final LocalDate visitDay;
```

Comment:
🤔**고민**🤔

SonarLint 에 따르면, 필드명은 클래스명과 중복되지 않는게 좋다고 합니다.
클래스명과 같다면 혼란을 줄 수 있다고 합니다.
조금 더 설명할 수 있는 이름으로 변경하는 걸 권장합니다.

Bad Practice
```java
public class Foo {
  private String foo;

  public String getFoo() { }
}

Foo foo = new Foo();
foo.getFoo() // what does this return?
```

Best Practice
```java
public class Foo {
  private String name;

  public String getName() { }
}

//...

Foo foo = new Foo();
foo.getName()
```

```suggestion
    private final LocalDate visitDate;
```

Reply:
이 부분에 대해서도 인지하고 있었지만, 고민이 많았던 로직이에요.
특히나, 일급 컬렉션 객체의 변수명을 짓는 과정은 꽤나 많은 고민을 하게 만들었어요.
여전히 변수명을 짓는데 오랜 시간을 할애하고, 고민하지만, 직관적인 변수명을 작명하는데는
어려움이 많은 것 같아요 :( 

소나린트도 사실 깔려는 있지만, 이번 과제에서 많이 활용하지는 않았던 것 같아요...!
린트를 준수하면서 조금 더 좋은 코드 작성을 하도록 노력해볼게요 :)